mobile/package.json

```diff
@@ -16,6 +16,7 @@
     "@react-navigation/native": "^5.5.0",
     "expo": "~37.0.3",
     "expo-updates": "~0.2.0",
+    "format": "^0.2.2",
```

Comment:
This doesn't look right. If you're trying to run Prettier, use `yarn format`, which will run the `format` script defined above on line 9 (which runs Prettier)